public/js/components/RightSidebar.css

```diff
@@ -1,13 +1,19 @@
 .right-sidebar {
+  display: flex;
+  flex-direction: column;
   flex: 1;
   white-space: nowrap;
 }
 
+.right-sidebar .accordion {
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
```

Comment:
May not need this `-webkit` property here, but might be useful in cases for outside Firefox?